packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesDevLauncherController.kt

```diff
@@ -114,26 +114,8 @@ class UpdatesDevLauncherController(
     context: Context,
     callback: UpdatesInterface.UpdateCallback
   ) {
-    if (updatesDirectory == null) {
-      callback.onFailure(updatesDirectoryException!!)
-      return
-    }
-
-    val newUpdatesConfiguration = when (UpdatesConfiguration.getUpdatesConfigurationValidationResult(context, configuration)) {
-      UpdatesConfigurationValidationResult.VALID -> UpdatesConfiguration(context, configuration)
-      UpdatesConfigurationValidationResult.INVALID_NOT_ENABLED -> {
-        callback.onFailure(Exception("Failed to load update: UpdatesConfiguration object is not enabled"))
-        return
-      }
-      UpdatesConfigurationValidationResult.INVALID_MISSING_URL -> {
-        callback.onFailure(Exception("Failed to load update: UpdatesConfiguration object must include a valid update URL"))
-        return
-      }
-      UpdatesConfigurationValidationResult.INVALID_MISSING_RUNTIME_VERSION -> {
-        callback.onFailure(Exception("Failed to load update: UpdatesConfiguration object must include a valid runtime version"))
-        return
-      }
-    }
+    val newUpdatesConfiguration = validateConfig(configuration, context)
```

Comment:
Previously this would call the callback with the exception? It throws now. I just want to make sure that that's expected?